console/src/helpers/Html.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import { env } from './localDev';
+/**
+ * Wrapper component containing HTML metadata and boilerplate tags.
+ * Used in server-side code only to wrap the string output of the
+ * rendered route component.
+ *
+ * The only thing this component doesn't (and can't) include is the
+ * HTML doctype declaration, which is added to the rendered output
+ * by the server.js file.
+ */
+
+type IAssets = {
+  [key in 'styles' | 'javascript']: {
+    [key: string]: string;
+  };
+};
+
+type HtmlProps = {
+  assets: IAssets;
+  component: React.ReactNode;
+  baseDomain: string;
+};
+
+export default function Html(props: HtmlProps) {
```

Comment:
We usually try use `const` instead `function` (in newer code), so to keep it consistent, could you change it to:
```ts
const Html = (...) => { ... }

export default Html;
```

And also as it's function component we can do:
`export const Html: React.FC<HtmlProps> = props => { ... }`